dts/bindings/espi/microchip,xec-espi-saf-v2.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2019 Intel Corporation
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip ESPI SAF version 2 controller
+
+compatible: "microchip,xec-espi-saf-v2"
+
+include: espi-controller.yaml
+
+properties:
+    reg:
+      description: mmio register space
+      required: true
+
+    girqs:
+      type: array
+      required: true
+      description: Array of encoded interrupt information
+
+    pcrs:
+      type: array
+      required: true
+      description: Array of eSPI PCR register index and bit position
+
+    poll-timeout:
+      type: int
+      description: poll flash busy timeout in 32KHz periods
+      required: false
+
+    poll-interval:
+      type: int
+      description: interval between flash busy poll in 20 ns units
+      required: false
+
+    consec-rd-timeout:
+      type: int
+      description: timeout after last read to resume supended operations in 20 ns units
+      required: false
+
+    sus-chk-delay:
+      type: int
+      description: hold off poll after suspend in 20 ns units
+      required: false
+
+    sus-rsm=interval:
```

Comment:
This looks like a typo; surely you didn't mean to use an `=` sign here; did you?

Reply:
yes, it is a typo

Reply:
fixed